src/session.ts

```diff
@@ -7,7 +7,10 @@ import { cookieName, cookieOptions } from './cookie.js';
 import { workos } from './workos.js';
 import { WORKOS_CLIENT_ID, WORKOS_COOKIE_PASSWORD } from './env-variables.js';
 import { getAuthorizationUrl } from './get-authorization-url.js';
-import { AccessToken, NoUserInfo, Session, UserInfo } from './interfaces.js';
+import { AccessToken, AuthkitMiddlewareAuth, NoUserInfo, Session, UserInfo } from './interfaces.js';
+
+import { parse, tokensToRegexp } from 'path-to-regexp';
+import { parse as parseURL } from 'url';
```

Comment:
Where does this dependency come from? It looks like we're relying on a transitive dep at the moment as we haven't added it to the package.json?

Reply:
It's part of the Node core library, but turns out it's actually deprecated so I'll refactor this.